Example/Example Swift/ComponentAPICoordinator.swift

```diff
@@ -679,12 +681,22 @@ extension ComponentAPICoordinator {
 
 extension ComponentAPICoordinator {
     
-    fileprivate func handleAnalysis(with extractions: [Extraction]) {
-        let payFive = ["paymentReference", "iban", "bic", "paymentReference", "amountToPay"]
-        let hasPayFive = extractions.filter { $0.name != nil ? payFive.contains($0.name!) : false }.count > 0
+    fileprivate func handleAnalysis(with extractionResult: ExtractionResult) {
+        let resultParameters = ["paymentRecipient", "iban", "bic", "paymentReference", "amountToPay"]
+        let hasExtractions = extractionResult.extractions.filter { resultParameters.contains($0.name ?? "no-name") }.count > 0
         
-        if hasPayFive {
-            showResultsTableScreen(withExtractions: extractions)
+        if hasExtractions {
+            
+            let images = self.pages.compactMap { $0.document.previewImage }
+            let extractions: [String: Extraction] = Dictionary(uniqueKeysWithValues: extractionResult.extractions.compactMap {
+                guard let name = $0.name else { return nil }
+                
+                return (name, $0)
+            })
+            
+            let result = AnalysisResult(extractions: extractions, lineItems: extractionResult.lineItems, images: images)
+            
+            showResultsTableScreen(with: result)
```

Comment:
You could also check here for `extractionResult.lineItems` and show the digital invoice screen, if it's present, no?

Reply:
Very well spotted and a few things fixed for Component API thanks to this 👍🏼

Reply:
Nice! 👍